pkg/gitreceive/build.go

```diff
@@ -256,8 +263,9 @@ func build(conf *Config, s3Client *s3.S3, builderKey, gitSha string) error {
 	if !usingDockerfile {
 		buildHook.Dockerfile = ""
 	} else {
+		json.Unmarshal([]byte("{}"), &procType)
```

Comment:
@bacongobbler I find it easy setting Proctype here . Makes sure that we are not messing up. But let me know if you want me to initialize Proctype. Its the same code but few steps above .